configure.ac

```diff
@@ -225,6 +225,20 @@ then
 fi
 AC_SUBST([ENABLED_ASM])
 
+# wolfEngine Options
+# This requires a FIPS source code bundle.
+AC_ARG_ENABLE([engine],
+    [AS_HELP_STRING([--enable-engine],[Enable wolfEngine options (default: disabled)])],
+    [ ENABLED_WOLFENGINE=$enableval ],
+    [ ENABLED_WOLFENGINE=no ]
+    )
+
+# wolfEngine for non-FIPS options
+AC_ARG_ENABLE([engine-without-fips],
```

Comment:
I'm curious why you decided to stick with having an extra option rather than making `--enable-engine` take an argument. Ex:

```
--enable-engine --> "yes", use FIPS.
--enable-engine=yes --> same thing
--enable-engine=fips --> same thing
--enable-engine=no --> don't enable engine options
--enable-engine=nofips -> enable engine non-FIPS options.
```

Reply:
Thank you for asking so diplomatically.  :smile:   I forgot.  Please stay turned.